docsteady/vcd.py

```diff
@@ -589,6 +632,7 @@ def check_acronyms(reqs):
 # 
 def vcdsql(comp, usr, pwd):
     global jst
```

Comment:
I don't think `global jst` and `global jpr` are needed here.  You should look for ways to avoid global variables altogether.